src/generators/testing/TestingGenerator.res

```diff
@@ -369,7 +369,7 @@ let script = () => {
 
   Generator.usePage("Color Test 2")
 
-  Generator.fillBackgroundColor("#ffffff")
+  Generator.fillBackgroundColor("#ffff00")
```

Comment:
Just a note that this background colour change and the one above break a couple of the colour tests I have.

The one up above should be black, and this one here should be white to get the tests to pass.

I need to clean up these tests a bit, I did them in a rush and they are a bit messy. Will try fix up soon.